build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -155,6 +155,8 @@ $_initializeTools
   var bootstrap = {
       hot\$onChildUpdate: function(childName, child) {
         if (childName === "$appModuleSource") {
+          // Clear static caches.
+          dart_sdk.dart.hotRestart();
```

Comment:
How long has `hotRestart()` existed in the SDK? Do we need to bump the minimum constraint in the pubspec?

Reply:
good question, @jmesserly do you know when this was added?

Reply:
I did some digging it looks like 2.1 was the stable release so I pinned to that